functions-methods.js

```diff
@@ -24,6 +72,25 @@
 
 /* Opdracht  3 */
 // Schrijf een functie genaamd checkEmailValidity, die een emailadres verwacht en checkt of het emailadres valide is. De functie returned true of false, afhankelijk van de uitkomst.
+
+const checkEmail = {
+    checkEmailValidity: function() {
+       const mailFormat = /^\w+([\.-]?\w+)*@\w+([\.-]?\w+)*(\.\w{2,3})+$/;
+       if(this.emailaddress2.match(mailFormat)) {
+            return true;
+        }
+        else {
+            return false;
+        }
```

Comment:
Je kan ook de uitkomst direct teruggeven
```suggestion
       return this.emailaddress2.match(mailFormat);
```